.github/workflows/smoke-test/smoke-test.sh

```diff
@@ -88,6 +88,17 @@ install_darwin() {
     cp -a "$srcApp" "$(dirname "$destApp")"
     xattr -d -r -s -v com.apple.quarantine "$destApp"
 
+    # Check that the image is compressed
+    local compressionRatio
+    compressionRatio="$(hdiutil imageinfo "$archiveName" \
```

Comment:
Does the `$archiveName` need to be escaped?

Reply:
No? It's a local variable from line 72, so it's the same shell session and it gets passed to `hdiutil` unmolested?